src/mlpack/methods/hoeffding_trees/hoeffding_tree_main.cpp

```diff
@@ -56,10 +56,10 @@ PARAM_STRING("output_model_file", "File to save trained tree to.", "M", "");
 
 PARAM_STRING("test_file", "File of testing data.", "T", "");
 PARAM_STRING("test_labels_file", "Labels of test data.", "L", "");
-PARAM_STRING("predictions_file", "File to output label predictions for test "
-    "data into.", "p", "");
-PARAM_STRING("probabilities_file", "In addition to predicting labels, provide "
-    "prediction probabilities in this file.", "P", "");
+PARAM_STRING("output_predictions_file", "File to output label predictions for"
+    "test data into.", "p", "");
+PARAM_STRING("output_probabilities_file", "In addition to predicting labels, "
+    "provide prediction probabilities in this file.", "P", "");
```

Comment:
I'm not sure, I could consider the command line executables and their options to be part of the public API. @zoq, @stereomatchingkiss, any thoughts on this one? If both of you agree that we don't violate our versioning policy here I think it is fine to make the change, otherwise maybe we should wait (or provide reverse compatibility options).